### ISSUE TITLE
[DEV] Map location data was not valid

### ISSUE THREAD

**Issue by mrwbsn** [PokemonGoF/PokemonGo-Bot#2424]
### Expected Behavior
Run normally and can snipe Pokemon like  #1992 

### Actual Behavior
with MoveToMapPokemon setting like this:
```
"config": {
          "address": "http://localhost:8007",
          "max_distance": 500,
          "min_time": 60,
          "prioritize_vips": true,
          "snipe": false,
          "update_map": true,
          "mode": "priority",
```

After login, bot will produce an error
```
Traceback (most recent call last):
  File "pokecli.py", line 446, in <module>
    main()
  File "pokecli.py", line 71, in main
    bot.tick()
  File "/home/mrwbsn/Workspaces/PokemonGo-Bot/pokemongo_bot/__init__.py", line 92, in tick
    if worker.work() == WorkerResult.RUNNING:
  File "/home/mrwbsn/Workspaces/PokemonGo-Bot/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 161, in work
    self.update_map_location()
  File "/home/mrwbsn/Workspaces/PokemonGo-Bot/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 107, in update_map_location
    return log.logger('Map location data was not valid', 'red')
NameError: global name 'log' is not defined
```
and if I disable update_map option to false, it will not produce error but a message:  `Map data is not valid`

### Steps to Reproduce
git pull, use config.json.map.example setting.

I have started my PokemonGoMap with `python -m SimpleHTTPServer 8007`

### Other Information
OS: Linux
Git Commit: 4a1f6493a78ce246aac81f243d2f562f22880529
Python Version: python 2.7.12

**Comment by h3k1ru** [comment]
Try to reach "http://localhost:8007/raw_data" by yourself, what do you see there?

**Comment by jbennett94** [comment]
Why was this closed? Having the same issue

**Comment by Fxwfran** [comment]
It's closed because it got fixed in #2053 . log.logger is a typo and should be changed to logger.log
